src/components/forms/DocumentUploadForm/shared/FileSelected/FileAsTitle.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <upload-title :title="title" />
+    <file-name :file-name="fileName" />
+  </div>
+</template>
+
+<script>
+import UploadTitle from '@/components/forms/DocumentUploadForm/shared/UploadTitle';
+import FileName from '@/components/forms/DocumentUploadForm/shared/FileName';
+
+export default {
+  name: 'FileAsTitle',
```

Comment:
I don't like `FileAsSomething`. May be `FileName`, `FileImage`, `FilePreview`? Why it is not possible?

Reply:
`FileName` already have in upper folder
`FilePreview` - it's not readable, that it's image
did not find any appropriate combination for more readable =(